login-workflow/src/translations/spanish.ts

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Mariana sent me some tweaks for the Spanish translations. I haven't updated them for the shared auth package yet, but we'll probably need to make them here too (we can do a follow-up for that, just letting you know).

Reply:
Yeah that sounds good to me; our translation string names aren't a one-to-one match, so it's a little tedious when trying to find the matching string between the two packages.